luna/pretrained_models/alexnet.py

```diff
@@ -141,7 +146,15 @@ def alex_net(weights_path=None, heatmap=False):
 
     model = Model(input=inputs, output=prediction)
 
-    if weights_path:
-        model.load_weights(weights_path)
+    weights_path = Path(str(Path.home()) + r"~\.keras\datasets\alexnet_weights.h5")
```

Comment:
I don't know if we can make these assumptions. Should we rather make the weights path a mandatory variable? Then one would have to specify it, and this would not be needed.

Reply:
Still valid.

Reply:
I don't know if that would work on every OS etc.